hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -586,9 +610,23 @@ extern "C" unsigned otbn_model_step(OtbnModel *model, svLogic start_i,
   if (!(status & RUNNING_BIT))
     return status;
 
+  bool edn_rnd_data_valid = false;
+  bool edn_urnd_data_valid = false;
+  uint32_t edn_rnd_data[8];
+
+  if (edn_rnd_data_valid_i) {
+    edn_rnd_data_valid = true;
+    set_rnd_data(edn_rnd_data, edn_rnd_data_i);
+  }
```

Comment:
It might be cleaner to pass `edn_rnd_data_i` straight through to `step_model` and then do the conversion there? That way, we can separate out the FSM stuff to do with starting and stopping from the "how to step the model" logic.

Reply:
Done, I've also passed through `err_bits` straight through so the Verilog DPI <-> C++ native types conversion is happening in the same place for all the arguments.